composer.json

```diff
@@ -21,26 +21,26 @@
     ],
     "require": {
         "php": ">=5.6",
```

Comment:
if the composer.json maintains PHP 5.6 as minimum, shouldn't we keep the travis-ci tests with this version ?

Reply:
I think is not worthy anymore to support PHP5.6 but, if we do, then yes, travis matrix should reflect this.

Reply:
Then which version of PHP will be supported? From PHP 7.2?

Reply:
As this bundle was "frozen" for a while, I suspect that go with 7.2 is not that crime. If we want, we could keep 7.*, don't know. As i "left" the (active) support for this bundle, I would not take a decision.

Reply:
If we follow Symfony requirements, I think we should have a version (3.X) compatible with Symfony 3 / PHP 5.6 and another (4.X) compatible with Symfony 4 / PHP 7.1

Reply:
@DonCallisto I get your point, but [packagists](https://packagist.org/packages/apy/datagrid-bundle/stats) show that there is about 200 installs of the bundle per day, and I guess that maybe "old" projects are using it.

I think we can drop PHP 5 support in the bundle as long as it is done in a next major version of it.

Reply:
No requirement for a major version from my standpoint as a minor bump just prevents someone to install it (so no BC break)-
It has sense only if you plan to provide support for both: in this case, yes, a new major should be the way to go.

Reply:
There's an error when installing with PHP 7.1 (see [travis](https://travis-ci.org/github/APY/APYDataGridBundle/jobs/633291707?utm_medium=notification&utm_source=github_status) ): 

Because in composer.json there is:

`"doctrine/mongodb-odm": "^2.0"`

Error message: 

`- doctrine/mongodb-odm 2.0.3 requires php ^7.2 -> your PHP version (7.1.27) does not satisfy that requirement.`

If I change it to "doctrine/mongodb-odm": "^1.0" then it will work, but this will downgrade some symfony packages to version 4.

Reply:
So, in order to keep all the compatibilities, we should require `php 7.2` as minimum php version (as I suspected). TBH I don't know if still keep a mongo support is a good idea, but as long as this bundle is really a mess of code, I would go with php7.2.
I mean, this bundle does not followed - in past - good practices both in code and in management. I think it would be hard to fix all the things at the same time.
Just my two cents, of course.

Reply:
Ok, so let me get this straight : the idea would be to have a new minor version of the bundle (3.3.0) with PHP 7.2 as a minimum version in `composer.json` and still a Symfony 3 compatibility ?

Reply:
Kind of

Reply:
I will send those changes, add check what travis says :D